Add Product Recipes Information Architecture Document (#2742)
mattermost/docs: source/administration/product-recipes-information-architecture.rst

Product Recipes Information Architecture
=========================================

> This document defined the product recipes information architecture. The product

Reviewer: ```suggestion
This document defines the product recipes information architecture. The product
```

> recipes might include information on

Reviewer: ```suggestion
recipes might include information on:
```

- Getting user traction for a product
- Product installation recipes
- Enablement of KPIs
- Enabling user requests

> Organising Product Recipes

Reviewer: ```suggestion
Organizing Product Recipes
```

--------------------------

> Defined are the things you should be following for organising product recipes.

Reviewer: ```suggestion
Below are the things you should follow for organizing product recipes.
```

- Write as per the document present in `administration/product-recipes-template.rst`.
- You should write all the product recipes in the `product-recipes` directory.
- If you create any new file you should add the file in the table of contents
  document.

> - Every product recipe should be linked with blog.

Reviewer: ```suggestion
- Every product recipe should be linked with a blog.
```

> - Different teams and members can use product recipe created, make sure you

Reviewer: ```suggestion
- Different teams and members can use the product recipes that were created. Make sure you
```
